CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Optionally can also contract the profiles of the original ``Equilibrium`` so tha
 - Allows Redl compute quantities to use SplineProfile
 - Updates Redl bootstrap current consistency tutorial to include a ``SplineProfile`` optimization
 - Adds automatically generated header file showing date the input file was created with `desc.vmec.VMECIO.write_vmec_input`
+- Adds max-J(dJ\_ds), direct omnigenity(dJ\_dalpha), adiabaric J compute, plotting and optimization objectives
```

Comment:
Refer to the actual plotting functions and objectives added, with the module path (like the line above `desc.vmec.VMECIO.write_vmec_input`)